### PR TITLE
[WebInterface] Add self update

### PR DESCRIPTION
**Pull request by Herklos**
WIP
image

Related to https://feedback.octobot.online/posts/6/add-an-octobot-update-button-in-webinterface

Related to https://github.com/Drakkar-Software/OctoBot/pull/1626

### CODE REVIEW

File: Services/Interfaces/web_interface/static/js/components/dashboard.js [hunk 1]
```diff
@@ -134,6 +134,20 @@ function get_announcements(){
     })
 }
 
+function get_version_upgrade(){
```
Comment (GuillaumeDSM) [inline Services/Interfaces/web_interface/static/js/components/dashboard.js:137]: 👍

File: Services/Interfaces/web_interface/templates/index.html [hunk 2]
```diff
@@ -2,6 +2,9 @@
 {% set active_page = "home" %}
 {% block body %}
     <div update-url="{{ url_for('api.announcements') }}" class="alert alert-danger text-center d-none" role="alert" id="annoncementsAlert"></div>
+    <div class="d-none alert alert-success text-center mt-2" role="alert">
+        <h5><i class="far fa-bell"></i> Good news ! OctoBot version <span update-url="{{ url_for('api.upgrade_version') }}" id="upgradeVersion"></span> is available. <button route="{{ url_for('commands', cmd='update') }}" type="button" class="btn btn-warning waves-effect">Upgrade now <i class="fas fa-cloud-download-alt"></i></button></h5>
```
Comment (GuillaumeDSM) [inline Services/Interfaces/web_interface/templates/index.html:6]: I'm wondering if we should make it take less space in mobile version 
image

Comment (GuillaumeDSM) [reply Services/Interfaces/web_interface/templates/index.html:6]: if we could fit it all in one line on mobile, it would be really nice

Comment (Herklos) [reply Services/Interfaces/web_interface/templates/index.html:6]: Right :+1:

Comment (Herklos) [reply Services/Interfaces/web_interface/templates/index.html:6]: Done 
image

Comment (Herklos) [reply Services/Interfaces/web_interface/templates/index.html:6]: Updated to one line :

image

**Review by GuillaumeDSM — APPROVED** [review]
💯

**Pull request merged**